src/IceRpc/Transports/Slic/Internal/SlicConnection.cs

```diff
@@ -598,6 +571,47 @@ async Task PerformDisposeAsync()
             _nextBidirectionalId = 0;
             _nextUnidirectionalId = 2;
         }
+
+        void SendPing(long payload)
+        {
+            try
+            {
+                WriteConnectionFrame(FrameType.Ping, new PingBody(payload).Encode);
+            }
+            catch (IceRpcException)
+            {
+                // Expected if the connection is closed.
+            }
+            catch (Exception exception)
+            {
+                Debug.Fail($"The sending of a Ping frame failed with an unexpected exception: {exception}");
+                throw;
+            }
+        }
+
+        void SendReadPing()
+        {
+            // ReadKeepAlive is no-op unless _pendingPongCount == 0 before the Increment below: we send a Ping only if
+            // there is no outstanding Pong.
```

Comment:
is `outstanding Pong.` correct here?

Reply:
Replaced by pending Pong, like the field name.